tensorflow/contrib/copy_graph/__init__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Functions for copying elements from one graph to another.
+
+@@copy_op_to_graph
+@@copy_variable_to_graph
+@@get_copied_op
```

Comment:
Can you add this module to gen_docs_combined.py? See the other contrib modules in there.